internal/model/plugin_test.go

```diff
@@ -73,6 +73,12 @@ func TestPluginsFromReader(t *testing.T) {
 				Signature:       "signature1",
 				ReleaseNotesURL: "https://github.com/mattermost/mattermost-plugin-demo/releases/v0.1.0",
 				Manifest:        &mattermostModel.Manifest{},
+				ArchBundles: ArchBundles{
+					LinuxAmd64: &ArchBundleMetadata{
```

Comment:
High-level, but would we consider a `LinuxDarwinWindowsAmd64` entry and make the the `Signature` and `DownloadURL` homogeneous?

Reply:
@lieut-data I'm not sure I understand. Do you mind showing me via an inline struct here?

Reply:
@lieut-data and I discussed offline, and decided the proposed change above is not required, and we will go forward using the existing `DownloadURL` and `Signature` for the default bundles.